recipes/glew/all/conanfile.py

```diff
@@ -75,6 +77,10 @@ def package(self):
         self.copy(pattern="LICENSE.txt", dst="licenses", src=self._source_subfolder)
 
     def package_info(self):
+        self.cpp_info.filenames["cmake_find_package_multi"] = "glew"
```

Comment:
Not for the `cmake_find_package` too?

Conan 2.0 will likely have only one `cmake_find_package` generator and the duplicity will no longer be needed, but until then, we need to define it for both generators 😞

Reply:
filename is uppercase for `cmake_find_package` (https://cmake.org/cmake/help/v3.1/module/FindGLEW.html) and lowercase for `cmake_find_package_multi`

Reply:
I see, these casing issues with CMake...